includes/UserAdd.php

```diff
@@ -102,7 +102,8 @@
 	}
 
 	$hash = password_hash( $_POST['passwd'] , PASSWORD_BCRYPT );
-
+	
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->